stacks.yaml

```diff
@@ -4028,6 +4028,40 @@ availableRuntimes:
     }
 
 
+   #####################
+   #   WildFly  14.0.1 #
+   #####################
+
+ - &wildfly-1401finalruntime
+   id: wildfly-1401finalruntime
+   name: WildFly 14.0.1 Final
+   groupId: org.wildfly
+   artifactId: wildfly-dist
+   version: 14.0.1.Final
+   license: *lgpl
+   url: http://wildfly.org/downloads/
+   downloadUrl: http://download.jboss.org/wildfly/14.0.1.Final/wildfly-14.0.1.Final.zip
+   boms:
+   defaultBom:
+   defaultArchetype: *wildfly-javaee7-webapp-archetype-latest
+   archetypes:
+    - *wildfly-javaee7-webapp-blank-archetype-latest
+    - *wildfly-javaee7-webapp-archetype-latest
+    - *wildfly-javaee7-webapp-ear-blank-archetype-latest
+    - *wildfly-javaee7-webapp-ear-archetype-latest
+    - *wildfly-html5-mobile-blank-archetype-latest
+    - *wildfly-html5-mobile-archetype-latest
+   labels: {
+        runtime-category: SERVER,
+        runtime-type: AS,
+        requiresLogModule: false,
+        pom: "https://github.com/wildfly/wildfly/blob/14.0.1.Final/pom.xml",
+        runtime-size: 179320953,
+        runtime-md5: 61a1279e0549089dc95519fced661999,
+        wtp-runtime-type: org.jboss.ide.eclipse.as.runtime.wildfly.141
```

Comment:
Can we change this to org.jboss.ide.eclipse.as.runtime.wildfly.140  ?

I realize you'll note that this isn't accurate, but, in jbosstools we already added the runtime type as 140 expecting the 14.0 download. So basically 14.x wildfly versions map to the 140 key in jbosstools.

Reply:
Ah okay. Should be fixed now.